scripts/xe-backup-metadata

```diff
@@ -39,6 +39,7 @@ function usage {
     echo " -k: Number of older backups to preserve (default: ${history_kept})"
     echo " -n: Just try to find a backup VDI and stop the script after that"
     echo " -f  Force backup even when less than 10% free capacity is left on the backup VDI"
+    echo " -y: Assume non-interactive mode and yes to all questions"
```

Comment:
missing hash of original commit in the commit message: 4fe8ffa0465abce5305b94a210db4bf8336155d1